### DISCUSSION TITLE
quantized TFLITE model support

### DISCUSSION THREAD

**Discussion by muzafferkal** [fastmachinelearning/hls4ml#650] [General]
Hi all,
I saw a PR about QONNX support and I was wondering what level of quantized TFLITE support there is.
Thanks

**Comment by vloncar** [comment]
Not supported and I am not aware of anyone working on it. FlatBuffers are not that easy to parse, and in most cases users have access to original model that can be quantized with QKeras rather than TFLite for better compatibility with hls4ml. We do welcome contributions if anyone is interested in contributing support for .tflite files.
